src/lib/Tab/LocationView/AuthorsTab.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\AdminUi\Tab\LocationView;
+
+use ArrayObject;
+use Ibexa\AdminUi\Specification\UserExists;
+use Ibexa\AdminUi\UI\Dataset\DatasetFactory;
+use Ibexa\Contracts\AdminUi\Tab\AbstractEventDispatchingTab;
+use Ibexa\Contracts\AdminUi\Tab\OrderedTabInterface;
+use Ibexa\Contracts\Core\Repository\SectionService;
+use Ibexa\Contracts\Core\Repository\UserService;
+use Ibexa\Contracts\Core\Repository\Values\Content\ContentInfo;
+use Ibexa\Contracts\Core\Repository\Values\Content\VersionInfo;
+use JMS\TranslationBundle\Annotation\Desc;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+use Twig\Environment;
+
+class AuthorsTab extends AbstractEventDispatchingTab implements OrderedTabInterface
+{
+    public const URI_FRAGMENT = 'ibexa-tab-location-view-authors';
+
+    protected UserService $userService;
+
+    protected SectionService $sectionService;
+
+    protected DatasetFactory $datasetFactory;
+
+    public function __construct(
+        Environment $twig,
+        TranslatorInterface $translator,
+        SectionService $sectionService,
+        UserService $userService,
+        DatasetFactory $datasetFactory,
+        EventDispatcherInterface $eventDispatcher
+    ) {
+        parent::__construct($twig, $translator, $eventDispatcher);
+
+        $this->sectionService = $sectionService;
+        $this->userService = $userService;
+        $this->datasetFactory = $datasetFactory;
+    }
+
+    /**
+     * @return string
+     */
+    public function getIdentifier(): string
+    {
+        return 'authors';
+    }
+
+    /**
+     * @return string
+     */
+    public function getName(): string
+    {
+        /** @Desc("authors") */
```

Comment:
```suggestion
        /** @Desc("Authors") */
```
It is weird because in `xliff` file we have `Authors`